.ci/README.md

```diff
@@ -89,6 +89,12 @@ If you are making changes to the containers, your changes will not apply until t
 
 Pausing the pipeline is done in the cloud console, by setting the downstream-builder trigger to disabled.  You can find that trigger [here](https://console.cloud.google.com/cloud-build/triggers/edit/f80a7496-b2f4-4980-a706-c5425a52045b?project=graphite-docker-images)
 
+### Downstream build job is not triggered by commits.
+This is rare but we've seen this happened before. In this case, we need to manually trigger a Cloud Build job by running 
+```
+gcloud builds triggers run build-downstreams --sha=<COMMIT_SHA> --substitutions=BRANCH_NAME=<BASE_BRANCH_NAME>
```

Comment:
Most folks will have their default project pointing to their personal dev project. Should we specify the CI project explicitly in this command?

Reply:
Absolutely, good point! I'll add it